python/tvm/relay/backend/graph_executor_factory.py

```diff
@@ -41,17 +41,18 @@ class GraphExecutorFactoryModule:
         The parameters of module
     """
 
-    def __init__(self, ir_mod, target, graph_json_str, libmod, libmod_name, params):
```

Comment:
So, this is the same as before. The "string" returned by `tvm.relay.build` is not necessarily a JSON file. It can be also a PrimFunc. That's why I am moving `graph_json_str` to be only `graph_str`

Reply:
I think we should not overload this as I've said in the other comments